### ISSUE TITLE
Collapsible blocks

### ISSUE THREAD

**Issue by Overload119** [notaapp/issues#18]
A-la https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab#a-collapsible-section-containing-code

This can help clean up large docs, and allow easily searching through them.

By default this would require support for something similar to Compact Links.

When the document is first rendered, it'll hide the section. 

ie.

A)
```
<details>
<summary>This is 1 line</summary>
</details>
```

Would render as:

[Expand] This is 1 line

If you click into this line it expands to A)

**Comment by astoilkov** [comment]
Interesting idea. What do you think of the possibility of collapsing headings instead of having a custom syntax? Like code editors expand/collapse of blocks.

**Comment by AntoineCheron** [comment]
+1 that would be a great feature.

Also, the ability to collapse list items would be great. It would very much help managing todo lists for example.

**Comment by astoilkov** [comment]
Yes. List items with child items are also a very good candidate. Thanks.

**Comment by erusev** [comment]
@AntoineCheron Thanks for joining the conversation! We've been researching "collapsing list items" for a while, and we're looking for more input on it. Could you share more about how it would help with managing todo lists (or other use cases that come to mind)? You can add comments #81 which @davistech just opened about this feature.
